airflow/kubernetes/pre_2_7_compatibility/pod_generator_deprecated.py

```diff
@@ -64,7 +64,7 @@ def make_safe_label_value(string):
 
     Valid label values must be 63 characters or less and must be empty or begin and
     end with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_),
-    dots (.), and alphanumerics between.
+    dots (.), and alpha-numbers between.
```

Comment:
This change sounds weird to me, why move away from alphanumerics?

Reply:
Yeah. It was just my 100th or so attempt to make the spell-checking pass for the provider with all the moved classes.

Reply:
Brought it back and added alphanumerics to spelling wordlist